tutorials/SpectroscopicDataReductionBasics/1-SpectroscopicTraceTutorial.ipynb

```diff
@@ -0,0 +1,1100 @@
+{
```

Comment:
<p>Change to text, not code comment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/508/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>done</p>